apps/hashdotai/docs/hash/getting-started/create-an-application.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Creating a simple application
+slug: hash/getting-started/creating-an-application
+objectId: ???
+description: Creating a simple application in HASH
+---
+
+# Create a simple application {#create-a-simple-application}
+
+Let’s explore how HASH works by building a simple application. We’re going to model our business’ projects in HASH and render our project data in a Table Block.
+
+1.  Sign up at [https://alpha.hash.ai/signup](https://alpha.hash.ai/signup).
+
+**Creating the “Project” Entity Type**
+
+1.  Click the + button next to TYPES in the left side-bar to create a new Type.
+1.  Name your Type “Project” and give it the description “A set of tasks which deliver a bigger business goal.” Click “Create new entity type”.
+
+**Creating Property Type and Data Types**
+
+1.  Click to “Add a property +” and add Text properties for:
+    1.  Title
+    1.  Description
+
+**Creating Link Types**
+
+1.  Click to “Add a link +” and add an “Owner” link type. Set the expected Entity Type as “User”.
+1.  Click “Create” in the top right corner to save this new Entity Type. \
+
+**Creating the “Sub-Task” Entity Type**
+
+1.  Click + next to Types in the side-bar again to create a 2nd Type
+1.  Call this one “Sub-task” and give it a description.
+1.  Again, add Text properties for Title and Description.
+1.  Click “Create” to save your type.
+
+**Creating the “Blocked by” Link Type**
+
+1.  Now use the menu under TYPES on the left to navigate back to your “Project” Type.
+1.  Click “Add a link +” and create a “Blocked by” Link Type, selecting the “Sub-Task” Type we just created as the expected link type.
+1.  Click to save your updated Entity Type.
+
+![alt_text](images/image2.png "image_tooltip")
```

Comment:
@jtewright Can images be sorted? They've not been included here, I don't believe.

Reply:
I can include the ones I mocked up - did you want to create new ones though?